src/feature/dirauth/dirauth_config.c

```diff
@@ -415,6 +409,12 @@ dirauth_options_pre_normalize(void *arg, char **msg_out)
                                   "AuthDirGuardBWGuarantee", msg_out) < 0)
     return -1;
 
+  if (options->MinUptimeHidServDirectoryV2 < 0) {
```

Comment:
This code is only run by the unit tests: config_parse_units() returned an unsigned 64-bit integer, and units_ok() checks that it's non-negative.